src/Portalum.Zvt/Parsers/BmpParser.cs

```diff
@@ -729,6 +736,20 @@ private bool ParseTerminalIdentifier(byte[] data, IResponse response)
             return false;
         }
 
+        private bool ParsePanData(byte[] data, IResponse response)
+        {
+            if (response is IResponseCardNumber typedResponse)
+            {
+                var cardNumber = BitConverter.ToString(data)
+                    .Replace("F", "")
```

Comment:
why is F,E,- replaced here?

Reply:
the raw output would be `EE-EE-EE-EE-EE-EE-EE-E0-32-4F`

* the `-` is an artifact from the Bit converter's output
* the `F` is a padding character for BCD encoding, since 0 cannot be used. As it would be a "0" then.
* the `E` character is the masking symbol, i have replaced it with `*` to match the output of the receipt. _This is debatable if it is a good idea._
![image](https://user-images.githubusercontent.com/7272789/188151270-e11d7d63-10af-463c-b52f-c948556987c7.png)

Reply:
What do you think about inserting an additional CardNumberRaw here?

Reply:
Im fine with either way (no post processing, or additional raw field). The library does hardly any post processing on the other fields so why should we do it here. What return type / format do you have in mind for the BCD encoding. String, with/without `-`?

Reply:
For me it also fits if we return it and the post processing is done in another place